cdk/bin/security-hq.ts

```diff
@@ -4,6 +4,8 @@
 import 'source-map-support/register';
 import { App } from '@aws-cdk/core';
 import { SecurityHQ } from '../lib/security-hq';
+import { SecurityVpc } from '../lib/security-vpc';
 
 const app = new App();
+new SecurityVpc(app, 'security-vpc', { stack: 'security ' });
```

Comment:
Is the extra space for `stack` intentional?

Reply:
oh no, not sure how that got added! Thanks.